lib/rules/no-multi-spaces.js

```diff
@@ -83,6 +83,34 @@ module.exports = {
             return false;
         }
 
+        /**
+         * Formats value of given comment token for error message by truncating its length.
+         * @param {ASTNode} token comment token
```

Comment:
Nit: I think tokens are technically not `ASTNodes`. I usually just put `{Token}` in the JSDoc comment, but I'm not sure if there's another convention that would be better.